burke_deannah_resources.md

```diff
@@ -0,0 +1,5 @@
+Coding/Turing/Git/Github resources:
```

Comment:
One trick with markdown files is to view them in their "rendered" state: https://github.com/deannahburke/mod0_student_resources/blob/main/burke_deannah_resources.md

When we write markdown, we should always be thinking about how it's going to look to the reader. No changes needed but practice on your own how you would make this more readable.